megamek/src/megamek/common/equipment/ArmorType.java

```diff
@@ -293,7 +307,7 @@ private static ArmorType createClanFerroFibrous() {
         armor.setInternalName("Clan Ferro-Fibrous");
         armor.addLookupName("Clan Ferro-Fibrous Armor");
         armor.addLookupName("Clan Ferro Fibre");
-        armor.tonnage = TONNAGE_VARIABLE;
```

Comment:
Is there a reason we're removing setting the tonnage?

Reply:
It's not used. Armor tonnage is determined by the number of points of armor on the unit and calculated by points per ton or weight per point depending on unit type.